tests/conftest.py

```diff
@@ -103,106 +94,89 @@ def gov(accounts):
     yield accounts.at("0x22eAe41c7Da367b9a15e942EB6227DF849Bb498C", force=True)
 
 
-@pytest.fixture(scope="module")
-def strategist_ms(accounts):
-    # like governance, but better
-    yield accounts.at("0xFB4464a18d18f3FF439680BBbCE659dB2806A187", force=True)
-
-
 @pytest.fixture(scope="module")
 def keeper(accounts):
-    yield accounts.at("0xC27DdC26F48724AD90E4d152940e4981af7Ed50d", force=True)
+    yield accounts.at("0x2757ae02f65db7ce8cf2b2261c58f07a0170e58e", force=True)
 
 
 @pytest.fixture(scope="module")
 def rewards(accounts):
-    yield accounts.at("0xC1c734c36a1Fb28502c48239995FC2b2d0031f81", force=True)
+    yield accounts.at("0x2757ae02f65db7ce8cf2b2261c58f07a0170e58e", force=True)
 
 
 @pytest.fixture(scope="module")
 def guardian(accounts):
-    yield accounts[2]
+    yield accounts.at("0x2757ae02f65db7ce8cf2b2261c58f07a0170e58e", force=True)
 
 
 @pytest.fixture(scope="module")
 def management(accounts):
-    yield accounts[3]
+    yield accounts.at("0xFB4464a18d18f3FF439680BBbCE659dB2806A187", force=True)
+
+
+@pytest.fixture(scope="module")
+def strategist_ms(accounts):
+    # like governance, but better
+    yield accounts.at("0xFB4464a18d18f3FF439680BBbCE659dB2806A187", force=True)
 
 
 @pytest.fixture(scope="module")
 def strategist(accounts):
-    strategist = accounts.at("0xC1c734c36a1Fb28502c48239995FC2b2d0031f81", force=True)
+    strategist = accounts.at("0x2757ae02f65db7ce8cf2b2261c58f07a0170e58e", force=True)
     yield strategist
 
 
-# # list any existing strategies here
-# @pytest.fixture(scope="module")
-# def LiveStrategy_1():
-#     yield Contract("0xC1810aa7F733269C39D640f240555d0A4ebF4264")
+# use this if your vault is already deployed
```

Comment:
for the tests, I ended up using a fresh vault as one or two tests were failing. These tests failed because the strategy used for testing purposes was a result of a migration from the old strategy as opposed to a freshly deployed one.